src/language/language_type.hbs.rs

```diff
@@ -484,9 +529,17 @@ pub fn get_filetype_from_shebang(file: &Path) -> Option<&'static str>
 
     let mut words = line.split_whitespace();
     match words.next() {
-        Some("#!/bin/sh") => Some("sh"),
-        Some("#!/bin/csh") => Some("csh"),
-        Some("#!/usr/bin/perl") => Some("pl"),
+        {{#each languages}}
+        {{~#each this.shebang_paths}}
+        Some("{{this}}")
+        {{~#unless @last}}
+        |
+        {{~/unless}}
+        {{~/each}}
```

Comment:
Same suggestion as above.
```suggestion
        {{~#each this.shebang_paths}}
        | Some("{{this}}")
        {{~/each}}
```